coding-concepts.js

```diff
@@ -10,46 +10,49 @@
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:it will add indigo to the array of words in colors
+// b) Verify and explain: it gave me the place holder of the 5th postiion within the array. 
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:this will be 10 because of the .lenght 
+// b) Verify and explain:answer ten becasue it counted all the characters within the cohort array from index0 placement
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:i think its gonna give me the 4th placement within the array of l i think.
+// b) Verify and explain: gave me the placement value of the following character which was "o"
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:i think it will give me "ruby" due to it being in the 1value of the index within the array.
+// b) Verify and explain: i go truby. so we called on languages and since we made index a const variable of 1, it gave us the value within the array instead of starting at the zero index
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer i think it will due an error but thats becasue i feel the quotes should be within the parenthesis after .toUpperCase
+
+// b) Verify and explain:it gave me an error but i dont know why after playing with the code 
+// console.log(weekendDays.toUpperCase())
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean", "undefined"]
-// console.log(typeof dataTypes.length)
+console.log(typeof dataTypes.length)
+
+// a) Your answer: i think it will give us the number of cahracters by each individual string
 
-// a) Your answer:
-// b) Verify and explain:
+// b) Verify and explain:it gave out number within the array. type of method i think it looked through the first array and found number. but am looking for the answer
```

Comment:
You did well on some of these!  Some were tricky. Remember the goal here is to understand the concept, so if you get and output that is not what you expected, research and explain why.  This will help you grasp the concepts better.  🧐